Source/Windows/ARM64EC/Module.cpp

```diff
@@ -183,6 +186,32 @@ void FillNtDllLUTs() {
   }
 }
 
+template<typename T>
+void WriteModuleRVA(HMODULE Module, LONG RVA, T Data) {
+  if (!RVA) {
```

Comment:
What's "RVA"?

Reply:
relative virtual address - see https://learn.microsoft.com/en-us/windows/win32/debug/pe-format

Reply:
Is it fair to assume this is common knowledge for anyone reading this code, or is it an acronym that could trivially be spelled out somewhere?

Reply:
Yeah, this is a frequently used term within wine